source/_docs/configuration/templating.markdown

```diff
@@ -183,6 +183,14 @@ The same thing can also be expressed as a filter:
 - `strptime(string, format)` parses a string based on a [format](https://docs.python.org/3.8/library/datetime.html#strftime-and-strptime-behavior) and returns a datetime object.
 - `relative_time` converts datetime object to its human-friendly "age" string. The age can be in second, minute, hour, day, month or year (but only the biggest unit is considered, e.g.,  if it's 2 days and 3 hours, "2 days" will be returned). Note that it only works for dates _in the past_.
 - `timedelta` returns a timedelta object and accepts the same arguments as the Python `datetime.timedelta` function -- days, seconds, microseconds, milliseconds, minutes, hours, weeks.
+
+{% raw %}
+```yaml
+# 77 minutes before curret time. 
+{{ now() - timedelta( hours = 1, minutes = 17 ) }} 
+```
+{% endraw %}
```

Comment:
This should be intended to stay inside the list, right?

```suggestion
   {% raw %}
   ```yaml
   # 77 minutes before curret time. 
   {{ now() - timedelta( hours = 1, minutes = 17 ) }} 
   ```
   {% endraw %}
```

Reply:
You're right. It should. Done.